deployments/deploy-ssh.sh

```diff
@@ -721,6 +721,8 @@ spec:
       value: "${S3_POLICY_SID}"
     - name: S3_ENCRYPTION_POLICY
       value: "${S3_ENCRYPTION_POLICY}"
+    - name: PX_NAMESPACE
+      value: "${PX_NAMESPACE}"
```

Comment:
Why is this required?

Reply:
If px is deployed in different namespace , restart stork will start failing .. so in case we have a setup while testing if px is deployed in different namespace we can pass the namespace and test or else we need to re-deploy the px

Reply:
There is already a function called `getExecPodNamespace` which gives you the namespace where portworx is running

Reply:
yes it makes sense , currently 3 tests uses DeletePodWithLabelInNamespace(getPXNamespace(), storkLabel)
Shall i go ahead and modify all?